webapp/lib/app/configurator/tags/view.dart

```diff
@@ -162,8 +165,18 @@ class ConfigureTagView extends TagView {
 
   ConfigureTagView(String tagText, String tagId, String groupId, TagStyle tagStyle, List<MenuItem> menuItems)
       : super(tagText, tagId, groupId: groupId, tagStyle: tagStyle, deletable: true, editable: true, menuItems: menuItems) {
-    var draggableTag = new dnd.Draggable(renderElement, avatarHandler: dnd.AvatarHandler.original(), draggingClass: 'tag__text');
-    draggableTag..onDragStart.listen((_) => dragInProgress = true)..onDragEnd.listen((_) => dragInProgress = false);
+    var draggableTag = new dnd.Draggable(renderElement, avatarHandler: dnd.AvatarHandler.original(), draggingClass: 'tag__text--dragging');
+    draggableTag
+      ..onDragStart.listen((_) => dragInProgress = true)
+      ..onDragEnd.listen((event) => dragInProgress = false)
```

Comment:
nit: ignore event parameter
```suggestion
      ..onDragEnd.listen((_) => dragInProgress = false)
```